cmd/libs/go2idl/client-gen/generators/generator_for_type.go

```diff
@@ -215,7 +215,7 @@ var listTemplate = `
 // List takes label and field selectors, and returns the list of $.type|publicPlural$ that match those selectors.
 func (c *$.type|privatePlural$) List(opts $.ListOptions|raw$) (result *$.type|raw$List, err error) {
 	result = &$.type|raw$List{}
-	err = c.client.Get().
+	err = c.client.GetRESTClient().Get().
```

Comment:
``newStructNamespaced`` could generate
```
// new$.type|publicPlural$ returns a $.type|publicPlural$
 func new$.type|publicPlural$(c *$.Group$Client, namespace string) *$.type|privatePlural$ {
 	return &$.type|privatePlural${
 		client: c.GetRESTClient(),
 		ns:     namespace,
 	}
 }
```

and ``structNamespaced`` with:

```
 // $.type|privatePlural$ implements $.type|public$Interface
 type $.type|privatePlural$ struct {
 	client $.RESTClient|raw$
 	ns     string
 }
```

Then, there would be no need to replace ``client.Get()`` with ``client.GetRESTClient().Get()``.